utils/compare_results.py

```diff
@@ -122,15 +120,27 @@ def get_rule_to_stig_dict(xml: ElementTree.ElementTree, is_disa: bool) -> dict:
 def get_results(xml: ElementTree.ElementTree) -> dict:
     rules = dict()
 
-    results_xml = xml.findall('.//xccdf-1.2:TestResult/xccdf-1.2:rule-result',
-                              ssg.constants.PREFIX_TO_NS)
+    results_xml = xml.findall('.//xccdf-1.2:rule-result', ssg.constants.PREFIX_TO_NS)
     for result in results_xml:
         idref = result.attrib['idref']
+        idref = idref.replace("xccdf_mil.disa.stig_rule_", "")
         rules[idref] = result.find('xccdf-1.2:result', ssg.constants.PREFIX_TO_NS).text
 
     return rules
 
 
+def get_identifiers(xml: ElementTree.ElementTree) -> dict:
+    rules = dict()
+
+    results_xml = xml.findall('.//xccdf-1.2:rule-result', ssg.constants.PREFIX_TO_NS)
```

Comment:
suddenly you use ssg.constants.PREFIX_TO_NS but you imported the PREFIX_TO_NS explicitly and you use it alone in the code above